eth-contracts/contracts/DelegateManager.sol

```diff
@@ -936,19 +930,19 @@ contract DelegateManager is InitializableV2 {
                and do validation
      * @param spFactory - reference to ServiceProviderFactory contract
      * @param _serviceProvider - address for which rewards are being claimed
-     * @return (totalBalanceInStaking, totalBalanceInSPFactory, totalActiveFunds, spLockedStake, totalRewards)
+     * @return (totalBalanceInStaking, totalBalanceInSPFactory, totalActiveFunds, spLockedStake, totalRewards, deployerCut)
```

Comment:
this change was to avoid the unnecessary second call to `getServiceProviderDetails` prior to this code fix